src/queries.ts

```diff
@@ -52,7 +52,7 @@ async function getStakedPSTHolders(): Promise<StakedPSTHolders> {
  */
 export async function getAllArDriveTransactionsWithin(range: HeightRange): Promise<GQLEdgeInterface[]> {
 	const cache = new GQLCache(range);
-	const nonCachedRanges = cache.getNonCachedRangesWithin().reverse();
+	const nonCachedRanges = cache.getNonCachedRangesWithin();
```

Comment:
Irrelevant change. It doesn't change the final behavior, it was only for me to see them run in ascending order.